frontend-build-pipeline.yml

```diff
@@ -6,8 +6,21 @@ trigger:
 pool:
   vmImage: ubuntu-latest
 
+parameters:
+- name: repoOrganization # the name of the repo's organization
```

Comment:
this is the sonarcloud org, not the github repo's org. they are 1-to-1 for us, but even for us there is a case difference between them.

Reply:
Yeah I got two mixed up earlier and didn't update the comment. I updated the comment to be more specific.